server.go

```diff
@@ -40,9 +42,20 @@ func NewServer(c *structs.Config) (*Server, error) {
 
 	showC := controllers.NewShowController(session, c)
 	//	getRouter.HandleFunc("/schedule/shows", showC.Get) // @TODO: Implement this
-	getRouter.HandleFunc("/schedule/shows/{id:[0-9]+}/", showC.GetShow)
-	getRouter.HandleFunc("/schedule/shows/timeslots/{id:[0-9]+}/", showC.GetTimeslot)
-	getRouter.HandleFunc("/schedule/shows/seasons/{id:[0-9]+}/", showC.GetSeason)
+	getRouter.HandleFunc("/schedule/shows/{id:[0-9]+}/", showC.GetShow).Name("show")
+	getRouter.HandleFunc("/schedule/shows/timeslots/{id:[0-9]+}/", showC.GetTimeslot).Name("timeslot")
+	getRouter.HandleFunc("/schedule/shows/seasons/{id:[0-9]+}/", showC.GetSeason).Name("season")
+
+	/* NOTE: NewScheduleWeekController assumes the following routes are
+	   installed BEFORE it is called:
+
+	   - 'timeslot' */
```

Comment:
If this is all it depends on, comment can probably be squished up

Reply:
Yeah, this is mainly in case other routes start referencing each other